lib/adrian/directory_queue.rb

```diff
@@ -69,9 +68,12 @@ def reserve(item)
     end
 
     def items
-      items = files.map { |file| wrap_item(file) }
-      items.reject! { |item| !item.exist? || filter?(item) }
-      items.sort_by(&:updated_at)
+      return @items if @items && @items.length > 0
+      @items = begin
+        items = files.map { |file| wrap_item(file) }
```

Comment:
I think that syntax for the `files.map` doesn't work because `wrap_item` is not a method of the members of `files`, which are strings (it belongs to `DirectoryQueue`).